### PR TITLE
[SYCL] Add workflow to test Joint Matrix with development version of IGC in pre-commit

### PR DESCRIPTION
**Pull request by YuriPlyakhin**

### CODE REVIEW

File: .github/workflows/sycl_containers.yaml [hunk 1]
```diff
@@ -123,11 +123,17 @@ jobs:
...
+            ghcr.io/${{ github.repository }}/ubuntu2204_intel_drivers_dev_igfx:latest-${{ github.sha }}
+            ghcr.io/${{ github.repository }}/ubuntu2204_intel_drivers_dev_igfx:latest
+          build-args: |
+            compute_runtime_tag=${{fromJson(steps.deps.outputs.deps).linux.compute_runtime.github_tag}}
+            igc_dev_hash=${{fromJson(steps.deps.outputs.deps).linux.igc_dev.github_hash}}
+            cm_tag=${{fromJson(steps.deps.outputs.deps).linux.cm.github_tag}}
+            level_zero_tag=${{fromJson(steps.deps.outputs.deps).linux.level_zero.github_tag}}
+            tbb_tag=${{fromJson(steps.deps.outputs.deps).linux.tbb.github_tag}}
+            fpgaemu_tag=${{fromJson(steps.deps.outputs.deps).linux.fpgaemu.github_tag}}
+            cpu_tag=${{fromJson(steps.deps.outputs.deps).linux.oclcpu.github_tag}}
```
Comment (aelovikov-intel) [inline .github/workflows/sycl_containers.yaml:139]: Do we really need these? If so, why?

Comment (bader) [reply .github/workflows/sycl_containers.yaml:139]: @YuriPlyakhin, could you please answer these questions if you plan to get this PR merged?

Comment (YuriPlyakhin) [reply .github/workflows/sycl_containers.yaml:139]: Sure, the PR is not ready yet

**Comment by jsji** [comment]
I will implement the workflow, split it into multiple PRs so that we can can review and test each change better.

1st PR posted: https://github.com/intel/llvm/pull/13184

**Comment by YuriPlyakhin** [comment]
Replaced with https://github.com/intel/llvm/pull/13516 and other PRs done by @jsji

**Pull request closed without merging**